briana-kc.md

```diff
@@ -0,0 +1,7 @@
+# Kansas City
+
+## Places to Eat
+- Joe's KC BBQ
```

Comment:
@hollenberry Yes! Worlds of Fun and Oceans of fun. I will add those in the next commit. 👍